.github/workflows/update-versions.yaml

```diff
@@ -40,10 +40,13 @@ jobs:
           OCP_IGNORED_VERSIONS: '["4.11","4.13"]'
           VERSION_FILE_PATH: "workflows/versions.json"
           TEST_TO_TRIGGER_FILE_PATH: "workflows/generatble-files/tests_to_trigger.txt"
+      - name: Generate timestamped branch name
+        run: echo "BRANCH_NAME=auto-pr-$(date +'%Y%m%d-%H%M%S')" >> $GITHUB_ENV
```

Comment:
Should we name it to explicitly tell it was about version updates? And the title as well.